### PR TITLE
Fixing sympify (evaluate = False) for Unary Operators - Issue #10560

### PR DESCRIPTION
**Pull request by sanyagithub**
`>>> sympify('4*-3', evaluate=False)`
`(-3)*4`

### CODE REVIEW

**Comment by smichr** [comment]
It's looking good so far. Thanks for working on this.

The next step is to add tests that cover the lines you have added. I would suggest testing the str version of an expression like those from the issue. And wherever the test is added, if it is part of an existing function, just label with `issue #10560` or else name a test function following the pattern of other tests that refer to the issue.

Also, if you put into your opening paragraph (OP) the phrase "fixes #10560" then the issue will be closed with this is merged. It will also create a link to the issue in the OP that others can click to go see what the issue was.

File: sympy/parsing/sympy_parser.py [hunk 1]
```diff
@@ -931,6 +931,11 @@ def visit_BinOp(self, node):
         if node.op.__class__ in self.operators:
             sympy_class = self.operators[node.op.__class__]
             right = self.visit(node.right)
+            left = self.visit(node.left)
+            if isinstance(node.left, ast.UnaryOp) and (isinstance(node.right, ast.UnaryOp) == 0) and sympy_class in ('Mul','Pow'):
+                temp = right
+                right = left
+                left = temp
```
Comment (jksuom) [inline sympy/parsing/sympy_parser.py]: There should be no need for ``temp``. Just assign ``left, right = right, left``.

Comment (sanyagithub) [inline sympy/parsing/sympy_parser.py]: Ok! I will do that! Thank you @jksuom :)

**Comment by sanyagithub** [comment]
@smichr Thank you for guiding me :) I have added the test cases in PR #10074

**Pull request closed without merging**